### PR TITLE
Add support for loading image from memory

### PR DESCRIPTION
**Pull request by vinothvino42**

### CODE REVIEW

File: lib/widget/image.dart [hunk 1]
```diff
@@ -154,6 +163,16 @@ class ImageState extends WidgetState<EnsembleImage> {
     return "${widget.controller.source}${str}timeStamp=$lastModifiedDateTime";
   }
 
+  Widget buildMemoryImage(Uint8List source) {
```
Comment (vusters) [inline lib/widget/image.dart:166]: does this work if it's png or svg?

**Comment by kmahmood74** [comment]
@vinothvino42 please respond to Vu's question

File: lib/widget/image.dart (hunk 1 above)
Comment (vinothvino42) [reply lib/widget/image.dart:166]: It's working fine for PNG and JPEG formats but it not working for SVG

**Comment by vinothvino42** [comment]
@vusters Now I added support for SVG too. Can you review it one more time?

**Review by snehmehta — CHANGES_REQUESTED** [review]

File: lib/widget/image.dart [hunk 2]
```diff
@@ -254,6 +278,20 @@ class ImageState extends WidgetState<EnsembleImage> {
   }
 
   bool isSvg() {
+    // Bytes Image
+    if (widget._controller.source is List<dynamic>) {
+      final Uint8List imageBytes =
+          Uint8List.fromList(widget._controller.source);
+
+      return imageBytes.length >= 5 &&
+          imageBytes[0] == 0x3C &&
```
[the remark this answers is not in the document]
Comment (vinothvino42) [reply lib/widget/image.dart]: Done

**Review by snehmehta — APPROVED** [review]

**Pull request merged**